commands/python/command/deploy.py

```diff
@@ -101,6 +101,11 @@ def deploy(
         # Values precedence is command < cluster rules < --set-override-values
         # Deploy command values
         values = {
+            "service.clusterCertificateHostname": cr.cluster_domain_name_suffix,
```

Comment:
it's not really a "certificate" hostname. I'm thinking of changing these to just `service.clusterHostname`  and `service.hostname`.